### ISSUE TITLE
[Tuning] GSuite.IsSuspiciousTag - Missing quote in test data email value

### ISSUE THREAD

**Issue by zestysoft** [panther-labs/panther-analysis#1880]
## Rule ID

GSuite.IsSuspiciousTag

## Description

The test case "Login Success But Flagged Suspicious" in `rules/gsuite_activityevent_rules/gsuite_is_suspicious_tag.yml` has a typo in the JSON test data - a missing opening quote for the email value:

```yaml
"actor": {
  "email": bobert@ext.runpanther.io"},
```

Should be:

```yaml
"actor": {
  "email": "bobert@ext.runpanther.io"},
```

Because YAML is flexible with unquoted strings, this doesn't cause a parsing error. Instead, YAML interprets `bobert@ext.runpanther.io"}` as the literal string value, including the trailing `"}` characters.

The test still passes, but the email field contains `bobert@ext.runpanther.io"}` instead of `bobert@ext.runpanther.io`. This can be seen in the test output where the title shows:

```
A suspicious action was reported for user [bobert@ext.runpanther.io"]
```

**Suggested Fix:** Add the missing opening quote to the email value and consider adding to `make test` to catch similar invalid data.
